news/_posts/2020-09-30-how-to-write-a-new-ui-test.md

```diff
@@ -0,0 +1,376 @@
+---
+layout: post
+title:  "Automated UI tests tutorial"
+subtitle: "Step by step - Adding a new PrestaShop UI test"
+date:  2020-09-30 12:00:00
+authors: [boubkerbribri]
+icon: icon-github-alt
+image: /assets/images/theme/meta-logo-build.png
+tags: [community, contribution, tests, automation]
+---
+
+One year ago, the QA team started to develop a new test framework, based on [Puppeteer](https://github.com/puppeteer/puppeteer), [Mocha](https://mochajs.org/), and [Chai](https://www.chaijs.com/).
+Since then, the tests coverage has been continually increasing and we recently improved our framework by switching to [Playwright](https://playwright.dev/).
+
+In this article, we will explain how to create a new UI (User Interface) test for the [PrestaShop core project](https://github.com/PrestaShop/PrestaShop).
+
+Before we start, you should definitely read about our stack and architecture in the [PrestaShop devdoc](https://devdocs.prestashop.com/1.7/testing/ui-tests/how-to-contribute-and-create-ui-tests/). 
+
+
+## I. Writing the scenario
```

Comment:
```suggestion
## Step 1: writing the scenario
```
simple titles are usually used on Build. maybe something like the suggestion is more clear?

Reply:
In the article, we talk about tests steps. So using it in another way could lead to a confusion. 
That's I used this type of titles.

Reply:
I agree, don't use "Step" anywhere but in the tutorial.